API/Program.cs

```diff
@@ -22,5 +25,23 @@ public static void Main(string[] args)
                 {
                     webBuilder.UseStartup<Startup>();
                 });
+        public static void RegisterGame(Game g)
+        {
+            games.Add(g.id, g);
+        }
+        public static Game GetGame(int id)
+        {
+            return games[id];
+        }
+        public static void DeleteGame(int id)
+        {
+            games.Remove(id);
+        }
+#if DEBUG
+        public static List<int> getGameIds()
+        {
+            return games.Keys.ToList();
+        }
+#endif
```

Comment:
Maybe we could extract this logic to a Singleton?

Reply:
resolved